protocol/communities/manager.go

```diff
@@ -574,92 +574,50 @@ func (m *Manager) CreateCommunity(request *requests.CreateCommunity, publish boo
 	return community, nil
 }
 
-func (m *Manager) CreateCommunityTokenPermission(request *requests.CreateCommunityTokenPermission) (*Community, *CommunityChanges, error) {
+func (m *Manager) CreateCommunityTokenPermission(request *requests.CreateCommunityTokenPermission) (*Community, *CommunityChanges, bool, error) {
```

Comment:
I think for now this change is fine, but maybe in the future we could consider adding this to the `CommunityChanges`